transaction-impl/src/main/java/Module.java

```diff
@@ -19,14 +19,5 @@
     protected void configure() {
         bindService(TransactionService.class, TransactionServiceImpl.class);
         bindClient(ItemService.class);
-
-        bind(TransactionServiceLifecycle.class).asEagerSingleton();
-    }
-
-    private static class TransactionServiceLifecycle {
```

Comment:
This is not necessary anymore.

Reply:
Is this the only `Module` where `gracefulShutdown` was still being used? I guess it was a leftover from the migration to 1.4.x...